opensciencegrid/gracc-apel/docker-run.sh

```diff
@@ -6,6 +6,9 @@ wrote=$(/usr/libexec/apel/apel_report.py "$@")
 echo "$wrote"
 apelfile=${wrote#wrote: }
 
+# Output the content of the apelfile before uploading, for debugging purposes
+cat $apelfile
```

Comment:
LGTM. Maybe add quotes, just in case, matching the line below.
```suggestion
cat "$apelfile"
```